ui-admin/src/util/formatQueryBuilderAsSearchExp.tsx

```diff
@@ -55,7 +55,11 @@ export const createEnrolleeSearchExpressionRuleProcessor = (facets: ExpressionSe
             if (isNumber(value)) {
               processedValue = value
             } else {
-              processedValue = parseNumber(value, { parseNumbers: true })
+              if (isString(value) && value.trim() === '') {
+                processedValue = 0
+              } else {
+                processedValue = parseNumber(value, { parseNumbers: true })
+              }
```

Comment:
totally random drive by fix, ran into this with A-T. if you are making a query in react query builder that uses a number and you clear the field, it creates an invalid query. this fixes it to default to 0 if nothing specified.